templates/vite/tsconfig.node.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Follow the other examples from the templates folder, this should use an inherited tsconfig base